plugins/Rockets/jsonSerialization.h

```diff
@@ -79,7 +80,43 @@ struct ModelTransferFunction
     size_t id;
     TransferFunction transferFunction;
 };
-}
+
+struct RPCDirectionalLight
```

Comment:
There aren't any static structures for the different camera types. Why do you want them for lights?

After continuing reading, I've realized that it must be for the RPC methods for adding lights, right? Check with @tribal-tec, because the same way these are not needed for changing the camera type and properties, they may neither be necessary for lights. The key could be in using PropertyObject instead of PropertyMap, but maybe there's no way to validate the properties in this case.